src/main/java/org/broadinstitute/consent/http/service/DataAccessReportsParser.java

```diff
@@ -8,6 +8,7 @@
 import org.broadinstitute.consent.http.util.DarUtil;
 import org.bson.Document;
 import java.io.*;
```

Comment:
Not your code, but we should avoid star imports. If you set up your IDE correctly, then when you optimize imports, these will go away.